lib/ansible/modules/packaging/os/apt_repository.py

```diff
@@ -109,10 +109,7 @@
 import sys
 import tempfile
 import json
```

Comment:
Looks good. Module JSON is also redundant here.

Reply:
Good catch!